.github/workflows/container.yml

```
@@ -18,96 +37,275 @@ permissions:
     runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        registry:
+          - name: ghcr
+            url: ghcr.io
+            username: ${{ github.actor }}
+            password: GITHUB_TOKEN
+          - name: dockerhub
+            url: docker.io
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)